src/main/java/at/hannibal2/skyhanni/features/event/hoppity/HoppityEventSummary.kt

```diff
@@ -406,10 +404,9 @@ object HoppityEventSummary {
             }
 
             put(HoppityStat.HOPPITY_RABBITS_BOUGHT) { statList, stats, _ ->
-                stats.mealsFound[HoppityEggType.BOUGHT]?.let {
-                    val rabbitFormat = StringUtils.pluralize(it, "Rabbit")
-                    statList.addStr("§7You bought §b$it §f$rabbitFormat §7from §aHoppity§7.")
-                }
```

Comment:
We don't want to do this. This would add a line every time independent of whether the stat was populated. Will be reverting.